base/system/chkdsk/lang/en-US.rc

```diff
@@ -0,0 +1,31 @@
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "\r\n\
+Usage: %s [drive:] [-F] [-V] [-R] [-C] \r\n\r\n\
+[drive:]    Specifies the drive to check.\r\n\
+-F          Fixes errors on the disk.\r\n\
+-V          Displays the full path of every file on the disk.\r\n\
+-R          Locates bad sectors and recovers readable information.\r\n\
+-C          Checks the drive only if it is dirty.\r\n\
+\r\n"
+
+    IDS_PERCENT_COMPL "%d percent completed.\r"
+    IDS_FILE_SYSTEM "The type of file system is %s.\n"
+    IDS_ABOUT "\n\
+Chkdskx v1.0.1 by Mark Russinovich\n\
+Systems Internals - http://www.sysinternals.com\n\
+ReactOS adaptation 1999 by Emanuele Aliberti\n\n"
```

Comment:
Why does this string use \n, but the IDS_USAGE use \r\n ?

Reply:
Indeed, the "\r" are superfluous.